config/samples/resources/cloudidsendpoint/cloudids_v1beta1_cloudnetwork-dep.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Google LLC
```

Comment:
Filename should be: `compute_v1beta1_computenetwork.yaml`. See naming convention: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#file-naming